draft-ietf-tls-esni.md

```diff
@@ -299,26 +342,30 @@ ClientEncryptedCH structure:
 ~~~~
 
 suite
-: The HpkeCipherSuite cipher suite used to encrypt ClientHelloInner. This MUST
-match a value provided in the corresponding ECHConfig.cipher_suites list.
+: The cipher suite used to encrypt ClientHelloInner. This MUST match a value
+provided in the corresponding `ECHConfig.cipher_suites` list.  [[TODO: Rename
+"suite" to "cipher_suite" to align with `ECHConfig`?]]
 
 record_digest
-: A cryptographic hash of the ECHConfig structure from which the ECH key was
+: A cryptographic hash of the `ECHConfig` structure from which the ECH key was
 obtained, i.e., from the first byte of "version" to the end of the structure.
 This hash is computed using the hash function associated with `suite`, i.e., the
-corresponding HPKE KDF algorithm hash.
+corresponding HPKE KDF algorithm hash. [[TODO: "record_digest" is a misnomer,
+since the value is not the hash of a record. What about something more
+descriptive, like "ech_config_id"?]]
 
 enc
 : The HPKE encapsulated key, used by servers to decrypt the corresponding
-encrypted_ch field.
+encrypted_ch field. [[TODO: A more descriptive name would be helpful. What about
```

Comment:
`enc` matches HPKE, so I think we should keep it as is.

Reply:
Ack, removing TODO